### PR TITLE
feat: add rector rule for format_size in 10.2

### PR DESCRIPTION
**Pull request by timohuisman**
## Description
The `format_size()` function in `common.inc` is deprecated in favour of using `\Drupal\Core\StringTranslation\ByteSizeMarkup::create()` that returns `TranslatableMarkup`

## Drupal.org issue
https://www.drupal.org/node/3413972

### CODE REVIEW

**Comment by timohuisman** [comment]
I guess the tests are failing because of #285

File: config/drupal-10/drupal-10.2-deprecations.php [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+declare(strict_types=1);
+
+use DrupalRector\Rector\Deprecation\FunctionToStaticRector;
+use DrupalRector\Rector\ValueObject\FunctionToStaticConfiguration;
+use Rector\Config\RectorConfig;
+use Rector\Symfony\Set\SymfonyLevelSetList;
+
+return static function (RectorConfig $rectorConfig): void {
+    $rectorConfig->sets([
+        SymfonyLevelSetList::UP_TO_SYMFONY_63,
```
Comment (bbrala) [inline config/drupal-10/drupal-10.2-deprecations.php]: This one is the same as 10.1, so we dont need this in here.

**Comment by bbrala** [comment]
Small little feedback, otherwise, great work!

**Comment by timohuisman** [comment]
Thanks for the review and the guidance @bbrala, I've resolved your feedback.

**Review by bbrala — APPROVED** [review]
Looking great! Thank you :)

**Comment by AlexSkrypnyk** [comment]
@bbrala 
This change does not have a fence for already replaced code. As a result - already processed code is reported as "needs fixing". Applying the fix again, results in another replacement, leading to unlimited requests to a replacement.

```
--- a/web/themes/contrib/civictheme/includes/utilities.inc
+++ b/web/themes/contrib/civictheme/includes/utilities.inc

-    'size' => DeprecationHelper::backwardsCompatibleCall(\Drupal::VERSION, '10.2.0', static fn() => format_size($file->getSize()), static fn(): TranslatableMarkup => ByteSizeMarkup::create($file->getSize())),
+    'size' => DeprecationHelper::backwardsCompatibleCall(\Drupal::VERSION, '10.2.0', static fn(): mixed => DeprecationHelper::backwardsCompatibleCall(\Drupal::VERSION, '10.2.0', static fn() => DeprecationHelper::backwardsCompatibleCall(\Drupal::VERSION, '10.2.0', static fn() => format_size($file->getSize()):
```

I think the CI needs to be updated to run the check twice to make sure that re-applying the rules on already fixed code does not trigger another need for a fix.

UPDATE: created issue in D.O. https://www.drupal.org/project/rector/issues/3414169

**Comment by bbrala** [comment]
Yeah, this is not this change though, but the overarching change which supports the bc calls. Added comment tot he issue. My collegue promised to have a look at whats broken, so might be able to fix it pretty soon.

**Pull request merged**